src/Interpreters/InterpreterCreateQuery.cpp

```diff
@@ -179,6 +179,7 @@ namespace ErrorCodes
     extern const int TOO_MANY_TABLES;
     extern const int TOO_MANY_DATABASES;
     extern const int THERE_IS_NO_COLUMN;
+    extern const int NONTEMPORARY_VIEW_ALREADY_EXISTS;
```

Comment:
```suggestion
```

Hm, I don't see this code in `ErrorCodes.cpp`, and it is not used here at all, please remove, it will not pass style check.